projects/rockylinux-8/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM rockylinux:8
+
+RUN \
+  yum update -y && \
+  yum install -y \
+    authd \
+    epel-release \
+    gcc-c++ \
+    gnupg \
+    make \
+    python3 \
+    python3-pip \
+    rsyslog \
+    sudo \
+    unixODBC-devel \
+    wget \
+    which \
+    diffutils \
+  && \
+  yum clean all && \
+  rm -rf /var/cache/yum /tmp/*
+
+# python 2 and 3 must be installed separately because yum will ignore/discard python2
+RUN \
+  yum check-update -q >/dev/null || { [ "$?" -eq 100 ] && yum update -y; } && \
+  yum install -y \
+    python3 \
+    python3-devel \
+    python3-pip \
+  && \
+  yum clean all && \
+  rm -rf /var/cache/yum /tmp/*
+
+RUN python3 -m pip install xmlrunner distro psutil pyodbc jsonschema requests
+
+#RUN \
+  #yum check-update -q >/dev/null || { [ "$?" -eq 100 ] && yum update -y; } && \
+  #yum install -y \
+    #python \
+    #python-devel \
+    #python-distro \
+    #python-pip \
+    #python-requests \
+    #python-jsonschema \
+    #python-psutil \
+  #&& \
+  #yum clean all && \
+  #rm -rf /var/cache/yum /tmp/*
```

Comment:
?

Reply:
Mmmmm holdover from almalinux:8, I think. I'll get rid of it.